src/nav/accounts.yml

```diff
@@ -15,6 +15,8 @@ pages:
             path: /docs/accounts/accounts-billing/account-setup/troubleshoot-new-relics-password-email-address-login-problems
           - title: Downgrade or cancel account
             path: /docs/accounts/accounts-billing/account-setup/downgradecancel-account
+          - title: Release notes
```

Comment:
Is this the correct place in the taxonomy? Or should we move it to live as a direct child of "Accounts and billing".

This works, just wanted to make sure it was the correct place.